### PR TITLE
 initialize project with components

### PR DESCRIPTION
**Pull request by Enning94**
## Bookstore: initialize project with components
In this Milestone, I:
- Initialized react app
- Added React Router and set two <Route>s and <Link>s for the app's navigation
- Created a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
- Created a separate component for this form (with inputs for a title and an author

### CODE REVIEW

File: src/Components/Book.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+// import './Styles/book.css';
```
Comment (Davidosky007) [inline src/Components/Book.js]: * Kindly remove all commented code from your code base as it makes your code base bulky and unprofessional 🙏

File: src/Components/Books.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Form from './Form';
+import Book from './Book';
+// import './Styles/book.css';
```
Comment (Davidosky007) [inline src/Components/Books.js]: * Same thing here Kindly remove all commented codes

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Enning94 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Book.js (hunk 1 above)
Comment (Enning94) [reply src/Components/Book.js]: Sure 👍 
Thanks

**Review by DeliceLydia — APPROVED** [review]
## Hi @Enning94, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**